cmd/ocm-backplane/script/listScripts_test.go

```diff
@@ -29,8 +27,8 @@ var _ = Describe("list script command", func() {
 		testToken     string
 		trueClusterId string
 		proxyUri      string
-		testKubeCfg   api.Config
-		testJobId     string
+		//testKubeCfg   api.Config
```

Comment:
Couldn't this be deleted instead of being commented out?

Reply:
done